### DISCUSSION TITLE
Heads up: Griffe v1 will use a single global "griffe" logger

### DISCUSSION THREAD

**Discussion by pawamoy** [PrefectHQ/prefect#14784] [General]
Hello! Latest version of Griffe, 0.48, deprecated its logger names. Since it's not possible to detect usage of logger names, we can't emit deprecation warnings, so here am I giving you a heads up :slightly_smiling_face: 

I see that you're disabling some Griffe loggers. Next version of Griffe, v1, will use a single name for all loggers (so, a global logger): `"griffe"`.

I suspect you don't want to disable *all* Griffe warning logs, and rather just the ones emitted in a few specific places. For this, v1 will let you import the Griffe logger and disable it temporarily:

```python
import griffe

with griffe.logger.disable():
    sections = griffe.parse_google(griffe.Docstring(o.__doc__))
```

Or similar, depending which logs you want to hide. If I understand correctly you also have utils to disable loggers. I'll let you see how to prepare for v1, and will try and answer any question you have regarding the upgrade to v1 😊

**Comment by cicdw** [comment]
Thanks for the proactive engagement on this @pawamoy !! We'll use this discussion if we run into any issues when we pick up the upgrade 👍
